### PR TITLE
Export ResponsiveMode

### LINKED ISSUE #9013: DropDown's responsiveMode prop enum type should exposed from 'office-ui-fabric-react'
### Environment Information 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 
 <Dropdown
          label={this.props.label}
          options={this.resolveOptions()}
          placeholder={this.props.placeholder}
          onRenderList={this.renderCalloutList}
          onRenderOption={this.props.customListElementRender}
          selectedKeys={this.props.selectedItems as string[]}
          multiSelect={this.props.multiSelect}
          onChange={this.props.onChange}
          disabled={this.props.isDisabled}
          styles={this.props.styles}
          errorMessage={this.props.errorMessage}
          onDismiss={this.onDropdownDismissed}
          calloutProps={calloutProps}
          responsiveMode={ResponsiveMode.large}
        />
#### Actual behavior:
 
ResponsiveMode is only available through  directly linkinig into lib

https://github.com/OfficeDev/office-ui-fabric-react/blob/acefdefb12eec6c495d5ffe25625b7da9d17cf78/packages/example-app-base/src/components/App/App.tsx#L7

This specifically crash create-react-app 
https://github.com/facebook/create-react-app

Jest test enviroment which compiled by node (thus need commonjs module)

if we include reference for the enum type for the number from lib-commonjs folder, we end packing both office-ui-fabrid-react/lib and office-ui-fabric-react/lib-commonjs
 
#### Expected behavior:

Since a component is using by a component  (in this case DropDown)
The prop type should be available from the same level, ie 'office-ui-fabric-react'

import {  ResponsiveMode } from 'office-ui-fabric-react'

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #9013
- [X] Include a change request file using `$ npm run change`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9018)

### CODE REVIEW

File: packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md [hunk 1]
```diff
@@ -3564,7 +3564,6 @@ export interface IDialogContentProps extends React.ClassAttributes<DialogContent
     draggableHeaderClassName?: string;
     isMultiline?: boolean;
     onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
-    // Warning: (ae-forgotten-export) The symbol "ResponsiveMode" needs to be exported by the entry point index.d.ts
```
Comment (jdhuntington) [inline packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: We should perhaps start breaking the build on warnings... this *was* called out. :)

Comment (dzearing) [reply packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: are there others?

Comment (dzearing) [reply packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: I am wondering if this is a new feature of api-extractor. First time I've seen this.

**Comment by dzearing** [comment]
@msft-github-bot merge in 2 hours.

File: packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md (hunk 1 above)
Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: IIRC the warning feature has existed but it is now capturing more symbols due to improvements in AE7. @natalieethell has more context.

Comment (natalieethell) [reply packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: Yeah Kevin's right! The warnings were around before, they just looked a little different:

```
// WARNING: The type "ICalloutState" needs to be exported by the package (e.g. added to index.ts)
```

And like Kevin said, AE7 is also now capturing more symbols.

Comment (natalieethell) [reply packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: Just checked and there are 38 of these warnings in office-ui-fabric-react.api.md

**Pull request merged**